tests/Microsoft.Bot.Connector.Tests/Authentication/EndorsementsRetrieverTests.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using Xunit;
```

Comment:
We really need to decide which test framework we are using.

(And yes my vote would be for xunit. It has Theories and I like the chai style BDD used here a lot.)